cicecore/cicedyn/infrastructure/io/io_pio2/ice_restart.F90

```diff
@@ -66,7 +66,7 @@ subroutine init_restart_read(ice_ic)
          filename = trim(ice_ic)
       else
          if (my_task == master_task) then
-            open(nu_rst_pointer,file=pointer_file)
+            open(nu_rst_pointer,file=pointer_file, status='old')
```

Comment:
We could / should probably do error checking on this open ? @apcraig ?

Anyway - its unrelated to this change

Reply:
That's exactly why I added the status="old", the fortran error message provided if the file doesn't exist should be adequate.